code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -143,7 +143,7 @@
 	desc = null //see examine()
 	icon_state = "nether"
 	max_integrity = 50
-	spawn_time = 600 //1 minute
+	spawn_time = 60 SECONDS
```

Comment:
I was split between having this be `60 SECONDS` versus `1 MINUTES` but I think `60 SECONDS` is more agreeable than the base type's (`/obj/structure/spawner`) definition of this variable that's being passed through in this PR.